src/Components/Panels/PanelMenu.tsx

```diff
@@ -139,6 +140,19 @@ export class PanelMenu extends SceneObjectBase<PanelMenuState> implements VizPan
         }),
       });
 
```

Comment:
Until we add it as a core part of the panel, you can also add it as a custom option https://github.com/grafana/logs-drilldown/blob/main/src/Components/ServiceScene/LogsPanelScene.tsx#L366